jdk/make/closed/autoconf/generated-configure.sh

```diff
@@ -4541,16 +4545,16 @@ VS_SDK_PLATFORM_NAME_2017=
 
 
 # Create a tool wrapper for use by cmake.
-# Consists of a shell script which wraps commands with an invocation of fixpath.
-# OPENJ9_GENERATE_TOOL_WRAPER(<name_of_wrapper>, <command_to_call>)
+# Consists of a shell script which wraps commands with an invocation of a wrapper command.
+# OPENJ9_GENERATE_TOOL_WRAPER(<name_of_output>, <name_of_wrapper>, <command_to_call>)
```

Comment:
This should be regenerated to reflect changes in `custom-hook.m4`.

Reply:
oops. thought I had